server/backend/api/app/schemas/__init__.py

```diff
@@ -1,46 +1,42 @@
 # --------------------------------- FAM Application --------------------------------- #
```

Comment:
Didn't change this file, but for some reason when I touched it my IDE rearranged the imports on save. Please ignore them.
Maybe we can remove the comment "#---#" from here, no need to group it just let IDE do the job...